### PR TITLE
Gradle updates

### PR DESCRIPTION
**Pull request by CodeCombiner**

### CODE REVIEW

File: android/CrashlyticsPlugin.java [hunk 1]
```diff
@@ -15,15 +15,17 @@
...
 
+
 public class CrashlyticsPlugin implements IPlugin {
 
   Activity _activity;
 
   public void onCreateApplication(Context applicationContext) {
-
+    logger.log("{crashlytics} registeriing crashlytics");
+    Fabric.with(applicationContext, new Crashlytics());
```
Comment (vivekb91) [inline android/CrashlyticsPlugin.java]: why we moved this to here?

Comment (CodeCombiner) [reply android/CrashlyticsPlugin.java]: This was recent update of Crashlytics and recommended to do it like this, as well it is useful to intercept errors which can arise after application instantiated but before activity is instantiated, these might be Android application components instances, plugins, etc.
https://stackoverflow.com/a/26539044/2988832

**Review by rampr — APPROVED** [review]

File: android/CrashlyticsPlugin.java [hunk 2]
```diff
@@ -15,15 +15,17 @@
 import com.tealeaf.logger;
 
 import com.crashlytics.android.Crashlytics;
-import com.crashlytics.android.core.CrashlyticsCore;
+import com.crashlytics.android.Crashlytics;
```
Comment (rampr) [inline android/CrashlyticsPlugin.java]: duplicate import

**Review by rampr — CHANGES_REQUESTED** [review]

**Comment by CodeCombiner** [comment]
Pushed fix

File: android/CrashlyticsPlugin.java [hunk 3]
```diff
@@ -15,15 +15,23 @@
...
 
+
 public class CrashlyticsPlugin implements IPlugin {
 
   Activity _activity;
 
   public void onCreateApplication(Context applicationContext) {
+    logger.log("{crashlytics} registeriing crashlytics");
+      Fabric.Builder fb = new Fabric.Builder(applicationContext)
+            .kits(new Crashlytics(), new CrashlyticsNdk());
```
Comment (rampr) [inline android/CrashlyticsPlugin.java]: Alex you had mentioned, CrashlyticsNdk will need 64 bit support.
So is it safe to go with this ?

Comment (CodeCombiner) [reply android/CrashlyticsPlugin.java]: You are right, it is not safe. Pushed commit

File: android/CrashlyticsPlugin.java [hunk 4]
```diff
@@ -15,15 +15,23 @@
...
 import io.fabric.sdk.android.Fabric;
 
+
 public class CrashlyticsPlugin implements IPlugin {
 
   Activity _activity;
 
   public void onCreateApplication(Context applicationContext) {
+    logger.log("{crashlytics} registeriing crashlytics");
+      Fabric.Builder fb = new Fabric.Builder(applicationContext)
```
Comment (rampr) [inline android/CrashlyticsPlugin.java:27]: indentation. but ignoring it for now.

**Pull request merged**